tests/gdb-tests/tests/heap/test_find_fake_fast.py

```diff
@@ -150,3 +150,11 @@ def test_find_fake_fast_command(start_binary):
     result = gdb.execute("find_fake_fast &target_address 0x100", to_string=True)
     check_no_results(result)
     gdb.execute("continue")
+
+    # setup_mem(0xAABBCCDD00000020, 0x8)
+    result = gdb.execute("find_fake_fast &target_address", to_string=True)
+    check_no_results(result)
+
+    result = gdb.execute("find_fake_fast &target_address -b", to_string=True)
```

Comment:
Nitpick but perhaps the full `--glibc-fastbin-bug` option here in place of `-b` could make it clearer what this is testing for.

Reply:
Updated